examples/sv1-mining-device/src/client.rs

```diff
@@ -25,7 +23,7 @@ const ADDR: &str = "127.0.0.1:34255";
 #[derive(Debug, Clone)]
 pub(crate) struct Client {
     client_id: u32,
-    extranonce1: Option<HexBytes>,
+    extranonce1: Option<U256<'static>>,
```

Comment:
This should be B032 not U256 cause extranonce can be at maximum 32 bytes but also less